.circleci/config.yml

```diff
@@ -118,9 +118,9 @@ jobs:
           command: |
             export PATH="$GOBIN:$PATH"
             make install
-            for pkg in $(go list github.com/cosmos/cosmos-sdk/... | grep -v /vendor/ | grep -v github.com/cosmos/cosmos-sdk/cmd/gaia/cli_test | circleci tests split --split-by=timings); do
```

Comment:
Figured I'd save some PR overhead. The `grep -v /vendor/` is unnecessary, go list excludes those now.